.github/workflows/daily-tests.yaml

```diff
@@ -80,6 +80,30 @@ jobs:
                   retention-days: 7
             - run: echo "This job's status is ${{ job.status }}."
 
+    gpu-tests:
+        runs-on: [self-hosted, linux, x64]
+        container: ghcr.io/gem5/gcn-gpu:latest
+        timeout-minutes: 300
+
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: develop
+            - name: Build VEGA_X86/gem5.opt
+              run: scons build/VEGA_X86/gem5.opt -j`nproc`
+
+            - name: Run Testlib GPU Tests
```

Comment:
Same question here

Reply:
My answer here is similar to the above but the only difference here is passing `--length=long` to state we want to run the Daily/Long tests found in the "tests/gem5/gpu" directory.